docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -1,21 +1,22 @@
 == Installing Jenkins plugins
-Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be installed by including them in Ansible playbook or using the Jenkins UI.
-
 NOTE: Plugins for Jenkins can be found at https://plugins.jenkins.io
 
+Digger will by default install required Jenkins plugins. If more Jenkins plugins are required, they can be installed by listing them in a task in Ansible playbook or using Jenkins UI.
+
 === Using Ansible Playbook
-The `jenkins_plugins` parameter can be overridden to include new plugins. The
-structure of a Jenkins plugin is as follows.
+https://github.com/aerogear/aerogear-digger-installer/blob/master/configure-buildfarm/defaults/main.yml[Digger Installer Ansible playbook^]. 
```

Comment:
Could we include the link somehow in the sentence rather than having it on it's own. Something like. The `jenkins_plugins` variable can be modified to include new plugins by editing it's value in the 'insert link here' defaults file

Reply:
resolved